sound/soc/sof/intel/hda.h

```diff
@@ -488,6 +488,7 @@ struct sof_intel_hda_stream {
  */
 int hda_dsp_probe(struct snd_sof_dev *sdev);
 int hda_dsp_remove(struct snd_sof_dev *sdev);
+int hda_dsp_shutdown(struct snd_sof_dev *sdev);
```

Comment:
Where does hda_dsp_shutdown be used?

Reply:
> Where does hda_dsp_shutdown be used?

Good catch, this was previous used but now should be removed.